workflow/rules/test_cases.smk

```diff
@@ -0,0 +1,34 @@
+rule test_non_cov2:
+    input:
+        pangolin=get_non_cov2_calls(from_caller="pangolin"),
+        kallisto=get_non_cov2_calls(from_caller="kallisto"),
+        call_plots=expand(
+            "results/test-cases/plots/strain-calls/non-cov2-{accession}.strains.{caller}.svg",
+            accession=get_non_cov2_accessions(),
+            caller=["pangolin", "kallisto"],
+        ),
```

Comment:
I guess this can be removed? It does not seem to be used in the script.

Reply:
Correct. I just used this to include the plots within the Snakemake report, that I shared over the matter most chat.